cpu/efm32/Makefile

```diff
@@ -1,6 +1,8 @@
 MODULE = cpu
 
 # add a list of subdirectories, that should also be build
-DIRS = periph families/$(EFM32_FAMILY) $(RIOTCPU)/cortexm_common
+DIRS += periph
+DIRS += families/$(CPU_FAM)
```

Comment:
I recall a discussion where I was misusing `CPU_FAM` for this purpose, and had renamed it to `EFM32_FAMILY`. I guess that changed?

Reply:
Oh, thats why it was there. I'll change it back since it is indeed not the right use.

Reply:
kept the old variable and exported it. Maybe we could move them all to separate dirst like `stm32`, not sure what is best.